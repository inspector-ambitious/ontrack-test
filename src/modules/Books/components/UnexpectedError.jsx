import React from "react";

const UnexpectedError = () => {
  return (
    <p className="mt-2 ml-4">
      An unexpected error occured. Please try again later...
    </p>
  );
};

export default UnexpectedError;
