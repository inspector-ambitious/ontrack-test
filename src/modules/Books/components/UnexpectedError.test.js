import React from "react";
import { shallow } from "enzyme";
import UnexpectedError from "./UnexpectedError";

describe("Books/components/UnexpectedError", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<UnexpectedError />);
    expect(wrapper).toMatchSnapshot();
  });
});
