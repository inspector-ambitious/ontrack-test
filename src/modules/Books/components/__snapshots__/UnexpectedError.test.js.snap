// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Books/components/UnexpectedError should render without crashing 1`] = `
<p
  className="mt-2 ml-4"
>
  An unexpected error occured. Please try again later...
</p>
`;
